example/android/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -3,4 +3,4 @@ distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-5.6.2-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-6.1.1-all.zip
```

Comment:
Same to above.

```groovy
distributionUrl=https\://services.gradle.org/distributions/gradle-6.5-all.zip
```

Reply:
@Kurogoma4D I have Updated Code
Kindly Review & merge the PR.